### PR TITLE
updates for the configure section

### PR DESCRIPTION
**Pull request by jonniebigodes**
This pull request starts addressing some of the feedback necessary for the migration of the documentation towards 6.0. In this particular case the configure section.

What was done:
- The links were updated.
- The user interface is now broken down into smaller sub pages:
   - Features and behavior
   - Theming
   - Storybook addons
  - Sidebar & URLS
  - Environment variables.

Feel free to provide feedback.

### CODE REVIEW

**Review by tmeasday — APPROVED** [review]
Fine by me. Looking forward to seeing it in a deploy preview.

**Comment by domyen** [comment]
lgtm, merge after resolving merge conflict

**Comment by jonniebigodes** [comment]
conflict is now resolved, going to merge as is, future updates will be made into follow up pull requests.

**Pull request merged**
